MAUI/Parallax-View/custom-controls.md

```diff
@@ -0,0 +1,145 @@
+                <parallax:SfParallaxView.Content>
+                    <Image x:Name="image" BackgroundColor="Transparent" HorizontalOptions="Fill" VerticalOptions="Fill" Aspect="AspectFill" />
+                </parallax:SfParallaxView.Content>
+            </parallax:SfParallaxView>
+            <local:CustomListView x:Name="listView" >
+                                  
+                   . . .
+
+            </local:CustomListView>
+            </Grid>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+          public class CustomListView : ListView, IParallaxView
+            {
+                public ListViewScrollingEventArgs scrollingEventArgs;
+                internal event EventHandler<ListViewScrollingEventArgs> CustomListViewScrolling;
+
+                private Size scrollableContentSize = new Size();
+                public Size ScrollableContentSize
+                {
+                    get
+                    {
+                        return this.scrollableContentSize;
+                    }
+                    set
+                    {
+                        this.scrollableContentSize = value;
+                        OnPropertyChanged("ScrollableContentSize");
+                    }
+                }
+
+                public event EventHandler<ParallaxScrollingEventArgs> Scrolling;
+
+                protected virtual void OnScrollChanged(ParallaxScrollingEventArgs e)
+                {
+                    Scrolling?.Invoke(this, e);
+                }
+
+
+                public CustomListView()
+                {
+                    this.scrollingEventArgs = new ListViewScrollingEventArgs(); // Need to initiate event args
+                    CustomListViewScrolling += CustomListView_CustomListViewScrolling; // Need to invoke custom scroll event
+                }
+
+                private void CustomListView_CustomListViewScrolling(object sender, ListViewScrollingEventArgs e)
+                {
+                    OnScrollChanged(new ParallaxScrollingEventArgs(e.ScrollX, e.ScrollY, false));
+                }
+            }
+
+            public class ListViewScrollingEventArgs : EventArgs
+            {
+                /// <summary>
+                /// Initializes a new instance of the <see cref="ListViewScrollingEventArgs" /> class.
+                /// </summary>
+                public ListViewScrollingEventArgs()
+                {
+                }
+
+                /// <summary>
+                /// Gets or sets ScrollX value.
+                /// </summary>
+                public double ScrollX { get; set; }
+
+                /// <summary>
+                /// Gets or sets ScrollY value.
+                /// </summary>
+                public double ScrollY { get; set; }
+            
+            }
+
+{% endhighlight %}
+
+{% endtabs %}
+
+By default, ParallaxView control supports [`.NET MAUI ScrollView`](). For custom controls you need to implement the [`IParallaxView`]() interface.
```

Comment:
By default, the ParallaxView control supports [`.NET MAUI ScrollView`](). For custom controls, you need to implement the [`IParallaxView`]() interface.